src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,173 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static User user;
+    private static User user2;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User(1L, "first_user", "password1", 21);
+        user2 = new User(2L, "second_user", "password2", 22);
+    }
+
+    @Test
+    void register_existingUser_notOk() {
+        Storage.people.add(user);
+        assertThrows(RegistrationException.class, () -> registrationService.register(user),
+                "Existing user must not be registered");
+    }
+
+    @Test
+    void register_notExistingUser_Ok() {
```

Comment:
```suggestion
    void register_notExistingUser_ok() {
```
use lowercase after _underscore everywhere